content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md

```diff
@@ -54,7 +54,7 @@ this example.
     cat << EOF > /etc/systemd/system/kubelet.service.d/20-etcd-service-manager.conf
     [Service]
     ExecStart=
-    ExecStart=/usr/bin/kubelet --address=127.0.0.1 --pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true
+    ExecStart=/usr/bin/kubelet --address=127.0.0.1 --pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true  --feature-gates=PodPriority=true
```

Comment:
there is an extra space before `--feature...`

but more importantly, there is no need to set `PodPriority` to true because it's already `true` since 1.11.
https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/

Reply:
Thanks, I will close this issue.